osquery/tables/system/windows/processes.cpp

```diff
@@ -95,15 +118,23 @@ void genProcess(long pid, QueryData& results_data) {
       r["gid"] = "-1";
       r["egid"] = "-1";
       r["sgid"] = "-1";
-
-      r["wired_size"] = "0";
-      r["resident_size"] = "0"; // Populate with WorkingSetSize (VT_BSTR)
-      r["phys_footprint"] = "0";
-
-      r["user_time"] = "0";
-      r["system_time"] = "0";
       r["start_time"] = "0";
 
+      // We pre-populate the info, in the event the API calls fail.
+      s = results[0].GetString("UserModeTime", sPlaceHolder);
+      long long llHolder;
+      osquery::safeStrtoll(sPlaceHolder, 10, llHolder);
+      r["user_time"] = BIGINT(llHolder / 10000000);
+      s = results[0].GetString("KernelModeTime", sPlaceHolder);
+      osquery::safeStrtoll(sPlaceHolder, 10, llHolder);
+      r["system_time"] = BIGINT(llHolder / 10000000);
+
+      s = results[0].GetLong("PagefileUsage", lPlaceHolder);
+      r["wired_size"] = BIGINT(lPlaceHolder);
```

Comment:
Is this result in bytes? If not can we use: `BIGINT(lPlaceHolder * 1024);` such that all the values have bytes as the unit.